src/TestRunner.js

```diff
@@ -63,9 +63,16 @@ export default class TestRunner {
       if (!this.shouldSendReport) return;
     }
 
+    const consoleReport = this.results.map(suite => {
```

Comment:
1. Could you use `flatMap`?
2. Both `flat` and `flatMap` require NodeJS 11. Does that conflict with React's requirements?
3. Would it be safer to use `reduce`?

Reply:
Approach changed - no rolling up of cases into suites.